README.md

```diff
@@ -1,19 +1,28 @@
 # Blueprint API
+While  ``blueprint api`` can be set up and used independently, it's recommended that setting up the local development 
+environment is done via the infrastructure repo: <https://github.com/CMSgov/blueprint_infra>.
 
-Please visit the infrastructure repo for setting up a local development environment at <https://github.com/CMSgov/blueprint_infra>
+If not utilizing the infrastructure repo to set up the project for development locally, the following are required:
+- python 3.10
+- postgres (or a running instance of postgres using the 
+[official docker image](https://github.com/docker-library/docs/tree/master/postgres#how-to-use-this-image))
+
+Create a database and superuser either with [initdb](https://www.postgresql.org/docs/14/app-initdb.html) or by supplying
+the necessary environment variables with using the postgres docker image.
```

Comment:
Praise: I appreciate this info for the docker-adverse crowd from one of their own. 😉

Reply:
lolz not docker-adverse, just too much in love with my debugging stuff (ask @amyd11, she's seen it in action)